client/blocks/utils.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import { getSetting } from '@woocommerce/settings';
+
+export const getStripeServerData = () => {
+	const stripeServerData = getSetting( 'stripe_data', null );
+
+	if ( ! stripeServerData ) {
+		throw new Error( 'Stripe initialization data is not available' );
+	}
+
+	return stripeServerData;
+};
```

Comment:
This function is now exclusively used for the Blocks integration, since the `getStripeServerData()` function in `stripe-utils` was causing a conflict with the UPE integration. See the commit message in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1825/commits/a9774f9f6745f8effaa9d3fb0df6d133248999fa for more details.

I decided to go with the same name since it serves the same purpose as the `getStripeServerData()` function in `stripe-utils`, but that may be confusing. Would it make more sense to call this function `getBlocksConfiguration()` (or similar) so it doesn't get confused with the other `getStripeServerData()` function?

Reply:
I like the idea of having different names... but in that case should we also rename the classic `getStripeServerData()` (the one in `client/stripe-utils/utils.js`) `getClassicConfiguration()`?

Reply:
I think so? But I'm worried about messing up other people's PRs that way, so I think we should open an issue for that 🤔 

For now I can change the Blocks integration function to `getBlocksConfiguration()` to reduce the ambiguity 👍

Reply:
Fixed in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1825/commits/a9e2c1955077f82046623692bdea54039c9c94a4